### DISCUSSION TITLE
User missing in command

### DISCUSSION THREAD

**Discussion by roelofz** [nextcloud/all-in-one#5413] [Questions] [answered]
In response to https://github.com/nextcloud/all-in-one/discussions/2810

@szaimen I think in the following command, the cron user is missing:

 */10 * * * * docker ps --format "{{.Names}}" | grep -q "^nextcloud-aio-nextcloud$" && docker exec --user www-data nextcloud-aio-nextcloud php occ preview:pre-generate

should be 

*/10 * * * *  **root** docker ps --format "{{.Names}}" | grep -q "^nextcloud-aio-nextcloud$" && docker exec --user www-data nextcloud-aio-nextcloud php occ preview:pre-generate

**Comment by szaimen** [comment, marked as answer]
Hi, since we recommend to use this command in the guide: `sudo crontab -u root -e`, I don't think the user is needed here

**Reply by roelofz** [reply]
Hi, 
I get it now, you use the user crontab of root, while I'm using the crontab in /etc, what I learned using in Ubuntu.

I do know it gave  me a problem, while the row is not loaded into the crontab without a username, when put in etc/crontab.

If I'm  the only one facing this, leave as it as it, it may however lead to more questions (a warning may be in place, up to you to decide).
